src/components/InputWithToggle/index.tsx

```diff
@@ -0,0 +1,89 @@
+import React, { useState } from 'react';
+import classNames from 'classnames';
+import Icons from '../Icons';
+
+export type TCheckboxType = 'default' | 'intermediate';
+export interface ICheckRadioProps {
+  size?: 'default' | 'large';
+  fieldName: string;
+  isSelected?: boolean;
+  disabled?: boolean;
+  children: React.ReactNode;
+  errorMessage?: string;
```

Comment:
The text of the error message isn't displayed anywhere, so It might be worth considering passing a bool prop instead like "invalid".

Reply:
Next iteration of this shows that we will have to show the message so I'll leave it like that for now.